assigner/commands/get.py

```diff
@@ -23,23 +27,76 @@ def get(conf, args):
     namespace = conf.namespace
     token = conf.gitlab_token
     semester = conf.semester
+    branch = args.branch
+    force = args.force
 
     roster = get_filtered_roster(conf.roster, args.section, args.student)
 
     path = os.path.join(hw_path, hw_name)
     os.makedirs(path, mode=0o700, exist_ok=True)
 
-    count = 0
-    for student in roster:
+    output = PrettyTable(["#", "Sec", "SID", "Name", "Change"], print_empty=False)
+    output.align["Name"] = "l"
+    output.align["Change"] = "l"
+
+    progress = tqdm()
+
+    for i,student in progress.enumerate(roster):
         username = student["username"]
         student_section = student["section"]
         full_name = StudentRepo.name(semester, student_section,
-                                     hw_name, username)
+                                    hw_name, username)
 
         try:
             repo = StudentRepo(host, namespace, full_name, token)
-            repo.clone_to(os.path.join(path, username))
-            count += 1
+            repo_dir = os.path.join(path, username)
```

Comment:
I think you can split here into another method without too many args... let me look:
somefunction(i + 1, repo, **student)

Am I missing something? That'll clean up the multiline variable definitions too maybe?

Reply:
I'm not sure it's worth it. Moving this would only make it more complicated.

Reply:
Yeah I'm loathe to split a function just for the sake of splitting a function.